### ISSUE TITLE
How do I disable VI mode?

### ISSUE THREAD

**Issue by ubershmekel** [sindresorhus/pure#560]
I'm on a mac and it seems Option-⬅ triggers VI mode. I just wanted to skip between different parts of the command I'm editing.

In general there's no documentation describing this vi mode, how to use it. I'd personally just like to remove it. Is there a way to disable the VI mode or at least change the keys that trigger it?

**Comment by mafredri** [comment]
Hi, Pure doesn't enable VI mode in any way, we simply support displaying VI-mode status, when it's activated by the user. If you look through your Zsh config, you should find a statement like `bindkey -v` somewhere, try removing that.

**Comment by ubershmekel** [comment]
There was no `bindkey -v` to be found. I found that in bash as well, my option-left shortcut wasn't skipping words backwards. Turns out that I had a setting in the terminal app that caused option-left to enter VI mode.

To fix this on macOS 10.15.6, I opened the terminal app's preferences, and had to set the bindings.

```
Option-left = \033b
Option-right = \033e
```

<img width="660" alt="Screen Shot 2020-09-02 at 5 22 55 PM" src="https://user-images.githubusercontent.com/77754/92051619-c7dc9280-ed41-11ea-925b-4996d101f32a.png">

I wrote this solution in https://stackoverflow.com/questions/81272/how-to-move-the-cursor-word-by-word-in-the-os-x-terminal/63715149#63715149 as well.

I guess pure is in the clear here so I'll close the issue. Thank you!
